.pre-commit-config.yaml

```diff
@@ -124,7 +124,7 @@ repos:
             fsspec,
             multidict,
             opentelemetry-instrumentation-asgi,
-            orjson,
+            msgspec@git+https://github.com/jcrist/msgspec.git,
```

Comment:
msgspec 0.10.0 has now been released, should be able to pin to the latest release now.